pkg/resources/jobs/runner.go

```diff
@@ -160,6 +161,15 @@ func NewRunnerJob(k6 *v1alpha1.K6, index int, testRunId, token string) (*batchv1
 						VolumeMounts:    script.VolumeMount(),
 						Ports:           ports,
 						EnvFrom:         k6.Spec.Runner.EnvFrom,
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path:   "/v1/status",
+									Port:   intstr.IntOrString{IntVal: 6565},
+									Scheme: "HTTP",
+								},
+							},
```

Comment:
This is an interesting addition! Strictly speaking this change is also out of scope of this PR :slightly_smiling_face: And there might be some small details in `/status` handler that might not fit how liveness probe should be implemented normally... I'll need to double-check this further. It might be best to move this change out to another PR; WDYT?

Reply:
See: https://github.com/grafana/k6-operator/pull/169#discussion_r1082884656